<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudgetTarget</class>
 <widget class="QWidget" name="BudgetTarget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Burget Target Team George</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="mainLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Weekly</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>801</width>
            <height>661</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QTreeWidget" name="treeWidget">
                   <property name="columnCount">
                    <number>3</number>
                   </property>
                   <column>
                    <property name="text">
                     <string notr="true">Category</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Amount</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Remaining</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QPushButton" name="Edit">
                   <property name="text">
                    <string>Edit Budget Item</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QTreeWidget" name="treeWidget_2">
               <property name="columnCount">
                <number>3</number>
               </property>
               <column>
                <property name="text">
                 <string>Category</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Amount</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Day</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPushButton" name="AddE">
                 <property name="text">
                  <string>Log Expense</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="DeleteE">
                 <property name="text">
                  <string>Delete Expense</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Monthly</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>821</width>
            <height>661</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QTreeWidget" name="treeWidget_3">
                   <property name="columnCount">
                    <number>3</number>
                   </property>
                   <column>
                    <property name="text">
                     <string notr="true">Category</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Amount</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Remaining</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QPushButton" name="Edit_2">
                   <property name="text">
                    <string>Edit Budget Item</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QTreeWidget" name="treeWidget_4">
               <property name="columnCount">
                <number>3</number>
               </property>
               <column>
                <property name="text">
                 <string>Category</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Amount</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Day</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QPushButton" name="AddE_2">
                 <property name="text">
                  <string>Log Expense</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="DeleteE_2">
                 <property name="text">
                  <string>Delete Expense</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Yearly</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>761</width>
            <height>661</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QTreeWidget" name="treeWidget_6">
                   <property name="columnCount">
                    <number>3</number>
                   </property>
                   <column>
                    <property name="text">
                     <string notr="true">Category</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Amount</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Remaining</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <widget class="QPushButton" name="Edit_4">
                   <property name="text">
                    <string>Edit Budget Item</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QTreeWidget" name="treeWidget_7">
               <property name="columnCount">
                <number>3</number>
               </property>
               <column>
                <property name="text">
                 <string>Category</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Amount</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Day</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QPushButton" name="AddE_3">
                 <property name="text">
                  <string>Log Expense</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="DeleteE_3">
                 <property name="text">
                  <string>Delete Expense</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>13</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
